### PR TITLE
Update check-release workflow

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -134,6 +134,7 @@ npm = [
 [tool.jupyter-releaser]
 skip = [
     "check-links",
+    "check-wheel-contents",
 ]
 
 [tool.jupyter-releaser.hooks]
```
Comment (jtpio) [inline pyproject.toml:140]: We'll also need to update the name of the `after-draft-release` hook.

Need to double check which one to use.

**Comment by jtpio** [comment]
This error:

```
dist/voila-0.5.0a1-py3-none-any.whl: W002: Wheel contains duplicate files:
```

Can probably be fixed like in the JupyterLab extension cookiecutter: https://github.com/jupyterlab/extension-cookiecutter-ts/blob/8dca7b37e31bc149ebaae29cc3f942aebf19a88d/%7B%7Bcookiecutter.python_name%7D%7D/pyproject.toml#L85

**Comment by martinRenou** [comment]
Thanks @jtpio !

**Comment by jtpio** [comment]
We could probably also remove the skip section here:

https://github.com/voila-dashboards/voila/blob/eab947ec004a596864dcf93a6eb93867610356bc/pyproject.toml#L134-L137

And add a `check-links.yml` workflow instead: https://github.com/jupyterlab/maintainer-tools#check-links

**Comment by martinRenou** [comment]
> We could probably also remove the skip section here:

Let's do that in a separate PR.

**Comment by jtpio** [comment]
Looks like there is also an old reference to `jupyter-packaging` that could be removed (since Voila is now using hatch).

**Pull request merged**